src/crypto/store/base.js

```diff
@@ -9,3 +9,23 @@
  *
  * @interface CryptoStore
  */
+
+/**
+ * Represents an outgoing room key request
+ *
+ * @typedef {Object} OutgoingRoomKeyRequest
+ *
+ * @property {string} requestId    unique id for this request
```

Comment:
The usual questions I ask whenever I see people add in IDs:
 - Is there a min length? Max length?
 - Can I use smiley poo? Spaces?